modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -1,101 +1,127 @@
+This uses the [TensorMechanics MasterAction](Modules/TensorMechanics/Master/index.md) to setup the StressDivergence Kernels because this simulation only includes real displacements.  A `Transient` executioner with a `ConstantDT` timestepper is used to sweep over frequencies where the time is substituted for the frequency.  The time is converted to the frequency dependent `Reaction` `rate` using a `ParsedFunction` and is applied to the `rate` using the `RealFunctionControl`.
 
-At this stage, we split the complex sytem of equations into two $\textbf{real}$ systems, which yields two real-valued systems that live on the same mesh. The strong form PDEs becomes as follows:
+In the next example, imaginary displacements will be introduced by including an absorbing boundary condition.  Imaginary displacement represent a phase shift in the problem and often result from damping.  
 
+# Uniaxial Beam Example
+
+The displacement field $U$ is a complex number.  In the previous example, only the real part of the displacement field was accounted for.  In this example, an absorbing boundary condition will introduce a coupling between the real and imaginary displacement fields.  This example uses the uniaxial beam shown in [cantilever] which is subjected to time-harmonic displacement on right and has an absorbing boundary condition on left.  The absorbing boundary condition is given by the Sommerfeld radiation condition as follows
+
+For the boundary condition, we apply the Sommerfeld radiation condition on the left, and a harmonic source (cosine) on the right as follows:
 \begin{equation}
-    -\nabla \cdot \hat \sigma_r- \omega^2  U_r = 0
-    \\
-    -\nabla \cdot \hat \sigma_i- \omega^2  U_i = 0 \label{eq8}
+    \sqrt E \, \nabla u= i\sqrt \rho \, \frac{\partial u}{\partial t} \, \qquad \text{at} \, x=0
+\label{eq6}
 \end{equation}
-with the following boundary conditions:
+The frequency domain version of [eq6] is
 \begin{equation}
-   \nabla U_r= -k U_i \quad ,\nabla U_i= k U_r\, \qquad \text{at} \, x=0,\\
-   U_r=U_i=0 \, \qquad \text{at} \, x=L, \label{eq9}
+    \frac{\partial U}{\partial x} U= ik\sqrt\frac{\rho}{E} U \qquad \text{at} \, x=0,
+    \label{eq7}
 \end{equation}
 
-Note that this decomposition is exact and there is no infromation missed while decompisition. Also note that, as a result of the radiation BCs on left, $U_r$ and $U_i$ are coupled, hence the two systems in [eq8].
-
-The following is how we define the kernels and the BCs. of this problem,
-
-!listing examples/wave_propagation/1D_elastic_wave_propagation.i block=Kernels id=listing1
+In general, $U$ is complex-valued function/variable in the form of $U_r+iU_i$ where $U_r$ and $U_i$ are the real and the imaginary part of $U$.  At this stage, we split the complex system of equations into two real valued systems that live on the same mesh. [eq3] for a complex valued system becomes
```

Comment:
As mentioned before, $U$ is complex-valued function/variable and can be written as $U_r+iU_i$ ...